resources/migrations/20190724113840-remove-application-form-fk.up.sql

```diff
@@ -0,0 +1,5 @@
+-- begin generating form IDs using form_template_id_seq instead of application_form_id_seq
+SELECT setval('form_template_id_seq', (SELECT nextval('application_form_id_seq')), false);
+--;;
+ALTER TABLE catalogue_item
+    DROP CONSTRAINT catalogue_item_ibfk_3;
```

Comment:
Hmm, should an analogous new constraint be added that links formId to id in form_template?

Reply:
That was added already earlier: https://github.com/CSCfi/rems/blob/master/resources/migrations/20190715105532-form-template-fk.up.sql

Reply:
Ok.